inc/datahandlers/post.php

```diff
@@ -708,14 +708,7 @@ function verify_prefix()
 						}
 					}
 
-					if($required['groups'] != "-1")
-					{
-						if(!is_member($required['groups'], array('usergroup' => $user['usergroup'], 'additionalgroups' => $user['additionalgroups'])))
-						{
-							$num_prefixes = true;
-						}
-					}
-					else
+					if(is_member($required['groups'], array('usergroup' => $user['usergroup'], 'additionalgroups' => $user['additionalgroups'])))
```

Comment:
I can change it, but I don't think it makes a big difference. This way clearly shows that groups are passed and the function only receives the values it needs.